Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067365): Gdk-CRITICAL **: 08:47:34.597: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068394): Gdk-CRITICAL **: 08:47:34.606: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069111): Gdk-CRITICAL **: 08:47:34.617: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067262): Gdk-CRITICAL **: 08:47:39.215: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068942): Gdk-CRITICAL **: 08:47:39.252: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067412): Gdk-CRITICAL **: 08:47:39.308: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068960): Gdk-CRITICAL **: 08:47:39.313: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068045): Gdk-CRITICAL **: 08:47:39.319: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068452): Gdk-CRITICAL **: 08:47:39.349: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068536): Gdk-CRITICAL **: 08:47:39.381: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068621): Gdk-CRITICAL **: 08:47:39.721: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068740): Gdk-CRITICAL **: 08:47:56.698: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067544): Gdk-CRITICAL **: 08:47:56.734: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067756): Gdk-CRITICAL **: 08:47:56.846: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067830): Gdk-CRITICAL **: 08:47:56.882: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067893): Gdk-CRITICAL **: 08:47:56.918: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067717): Gdk-CRITICAL **: 08:47:56.938: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068142): Gdk-CRITICAL **: 08:47:56.946: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068412): Gdk-CRITICAL **: 08:47:56.974: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068460): Gdk-CRITICAL **: 08:47:56.978: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067827): Gdk-CRITICAL **: 08:47:56.982: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067804): Gdk-CRITICAL **: 08:47:56.994: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067752): Gdk-CRITICAL **: 08:47:57.010: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068603): Gdk-CRITICAL **: 08:47:57.014: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067736): Gdk-CRITICAL **: 08:47:57.018: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069092): Gdk-CRITICAL **: 08:47:57.026: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067259): Gdk-CRITICAL **: 08:47:57.077: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067267): Gdk-CRITICAL **: 08:47:57.081: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068932): Gdk-CRITICAL **: 08:47:57.085: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067278): Gdk-CRITICAL **: 08:47:57.093: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069120): Gdk-CRITICAL **: 08:47:57.113: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067920): Gdk-CRITICAL **: 08:47:57.117: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067979): Gdk-CRITICAL **: 08:47:57.120: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068194): Gdk-CRITICAL **: 08:47:57.133: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068211): Gdk-CRITICAL **: 08:47:57.153: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068761): Gdk-CRITICAL **: 08:47:57.156: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067464): Gdk-CRITICAL **: 08:47:57.176: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067617): Gdk-CRITICAL **: 08:47:57.200: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069091): Gdk-CRITICAL **: 08:47:57.245: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068424): Gdk-CRITICAL **: 08:47:57.292: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069113): Gdk-CRITICAL **: 08:47:57.301: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068012): Gdk-CRITICAL **: 08:47:57.304: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068946): Gdk-CRITICAL **: 08:47:57.317: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069125): Gdk-CRITICAL **: 08:47:57.346: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068179): Gdk-CRITICAL **: 08:47:57.348: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067579): Gdk-CRITICAL **: 08:47:57.356: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068136): Gdk-CRITICAL **: 08:47:57.363: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068492): Gdk-CRITICAL **: 08:47:57.382: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068382): Gdk-CRITICAL **: 08:47:57.389: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068997): Gdk-CRITICAL **: 08:47:57.418: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068954): Gdk-CRITICAL **: 08:47:57.425: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067937): Gdk-CRITICAL **: 08:47:57.430: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069136): Gdk-CRITICAL **: 08:47:57.434: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068775): Gdk-CRITICAL **: 08:47:57.438: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067496): Gdk-CRITICAL **: 08:47:57.441: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068003): Gdk-CRITICAL **: 08:47:57.445: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067670): Gdk-CRITICAL **: 08:47:57.450: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068169): Gdk-CRITICAL **: 08:47:57.453: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068799): Gdk-CRITICAL **: 08:47:57.466: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068202): Gdk-CRITICAL **: 08:47:57.469: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068772): Gdk-CRITICAL **: 08:47:57.493: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067647): Gdk-CRITICAL **: 08:47:57.521: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067287): Gdk-CRITICAL **: 08:47:57.524: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067255): Gdk-CRITICAL **: 08:47:57.536: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068667): Gdk-CRITICAL **: 08:47:57.540: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069129): Gdk-CRITICAL **: 08:47:57.556: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4068632): Gdk-CRITICAL **: 08:47:57.559: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067960): Gdk-CRITICAL **: 08:47:57.565: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4068716): Gdk-CRITICAL **: 08:47:57.567: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068784): Gdk-CRITICAL **: 08:47:57.572: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4068643): Gdk-CRITICAL **: 08:47:57.575: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068558): Gdk-CRITICAL **: 08:47:57.581: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4068611): Gdk-CRITICAL **: 08:47:57.583: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068984): Gdk-CRITICAL **: 08:47:57.589: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4068710): Gdk-CRITICAL **: 08:47:57.591: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068218): Gdk-CRITICAL **: 08:47:57.600: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4068970): Gdk-CRITICAL **: 08:47:57.603: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068029): Gdk-CRITICAL **: 08:47:57.608: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068924): Gdk-CRITICAL **: 08:47:57.612: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068433): Gdk-CRITICAL **: 08:47:57.616: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068471): Gdk-CRITICAL **: 08:47:57.620: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068438): Gdk-CRITICAL **: 08:47:57.624: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068752): Gdk-CRITICAL **: 08:47:57.628: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068981): Gdk-CRITICAL **: 08:47:57.632: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068157): Gdk-CRITICAL **: 08:47:57.636: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067281): Gdk-CRITICAL **: 08:47:57.640: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067341): Gdk-CRITICAL **: 08:47:57.644: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4069097): Gdk-CRITICAL **: 08:47:57.652: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4068548): Gdk-CRITICAL **: 08:47:57.655: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067993): Gdk-CRITICAL **: 08:47:57.665: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067845): Gdk-CRITICAL **: 08:47:57.668: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068130): Gdk-CRITICAL **: 08:47:57.672: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4067252): Gdk-CRITICAL **: 08:47:57.675: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067772): Gdk-CRITICAL **: 08:47:57.684: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068615): Gdk-CRITICAL **: 08:47:57.693: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067275): Gdk-CRITICAL **: 08:47:57.705: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4068731): Gdk-CRITICAL **: 08:47:57.708: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4067780): Gdk-CRITICAL **: 08:47:57.715: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:4067265): Gdk-CRITICAL **: 08:47:57.759: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:4067271): Gdk-CRITICAL **: 08:47:57.796: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
..................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
[ 1777 10000]
.
[ 1764 10000]
.
[ 1724 10000]
.
[ 1741 10000]
.
[ 1764 10000]
.
[ 1738 10000]
.
[ 1725 10000]
.
[ 1712 10000]
.
[ 1704 10000]
.
[ 1725 10000]
.
[ 1719 10000]
.
[ 1678 10000]
.
[ 1750 10000]
.
[ 1711 10000]
.
[ 1736 10000]
.
[ 1755 10000]
.
[ 1796 10000]
.
[ 1728 10000]
.
[ 1706 10000]
.
[ 1732 10000]
.
[ 1718 10000]
.
[ 1680 10000]
.
[ 1706 10000]
.
[ 1724 10000]
.
[ 1687 10000]
.
[ 1735 10000]
.
[ 1691 10000]
.
[ 1674 10000]
.
[ 1703 10000]
.
[ 1683 10000]
.
[ 1683 10000]
.
[ 1732 10000]
.
[ 1792 10000]
.
[ 1707 10000]
.
[ 1695 10000]
.
[ 1705 10000]
.
[ 1693 10000]
.
[ 1742 10000]
.
[ 1751 10000]
.
[ 1674 10000]
.
[ 1738 10000]
.
[ 1696 10000]
.
[ 1745 10000]
.
[ 1735 10000]
.
[ 1688 10000]
.
[ 1723 10000]
.
[ 1708 10000]
.
[ 1667 10000]
.
[ 1702 10000]
.
[ 1735 10000]
.
[1690 9998]
.
[ 1756 10000]
.
[ 1717 10000]
.
[ 1715 10000]
.
[ 1766 10000]
.
[ 1698 10000]
.
[ 1702 10000]
.
[ 1709 10000]
.
[ 1725 10000]
.
[ 1692 10000]
.
[ 1709 10000]
.
[ 1729 10000]
.
[ 1738 10000]
.
[ 1748 10000]
.
[ 1748 10000]
.
[ 1754 10000]
.
[ 1708 10000]
.
[ 1677 10000]
.
[ 1711 10000]
.
[ 1689 10000]
.
[ 1718 10000]
.
[ 1681 10000]
...
[ 1686 10000]
..................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
Data Dump...
..
Data Dump...
.....
Data Dump...
..
Data Dump...
..
Data Dump...
...
Data Dump...

Data Dump...
.
Data Dump...

Data Dump...

Data Dump...

Data Dump...
.
Data Dump...
...
Data Dump...

Data Dump...

Data Dump...
.
Data Dump...
.
Data Dump...
.....
Data Dump...
....
Data Dump...

Data Dump...
.
Data Dump...
....
Data Dump...
.
Data Dump...
.
Data Dump...
......
Data Dump...
.
Data Dump...
..
Data Dump...
...
Data Dump...
..
Data Dump...
.
Data Dump...
.
Data Dump...

Data Dump...

Data Dump...
.
Data Dump...
..
[  35 9888]
.
Data Dump...

[  46 9625]
.
Data Dump...
.
Data Dump...
.
[  46 9520]
.
Data Dump...

[  43 9762]
..
[  35 9672]
.
Data Dump...

[  35 9722]
.
Data Dump...

Data Dump...
...
[  45 9844]
.
Data Dump...
.
Data Dump...
..
Data Dump...
.
[  32 9584]
..
[  37 9545]
.....
[  46 9574]
.
[  39 9366]
.
Data Dump...
..
[  41 9146]
.
[  45 9608]
..
[  37 9728]
.
[  44 9638]
.
[  34 9326]
.
Data Dump...
..
Data Dump...
..
[  30 9525]
......
[  41 9700]
.
[  49 9523]
.
[  39 9756]
...
[  40 9723]
.
Data Dump...
.
[  42 9663]
......
[  36 8420]
.
[  47 8353]
...............................................................
[  67 7038]
.....................................................................................................................................
[  35 1768]
.
[  45 1743]
.
[  35 1758]
.
[  43 1664]
.
[  35 1623]
.
[  37 1626]
.....
[  46 1431]
....
[  37 1896]
.
[  45 1037]
..
[  44 1802]
.
[  45 1682]
.....
[ 32 729]
.
[  46 1146]
.......
[ 39 924]
..
[ 40 975]
.
[  49 1226]
..
[ 39 967]
.
[ 34 754]
..
[  40 1196]
...
[  42 1068]
..
[ 41 511]
.
[ 30 679]
...................................
[ 36 400]
..
[ 46 733]
......................................................................................................
[ 46 893]
...
Data Dump...
...
Data Dump...

Data Dump...

Data Dump...
.........
Data Dump...
....
Data Dump...
....
Data Dump...
..
Data Dump...
...
Data Dump...

Data Dump...
....
Data Dump...
..
Data Dump...

Data Dump...
...........
Data Dump...
.
Data Dump...

Data Dump...

Data Dump...

Data Dump...

Data Dump...
...
Data Dump...

Data Dump...

Data Dump...
.....
Data Dump...

Data Dump...
.....
Data Dump...
High 1 0
Done
slurmstepd-phcompute010: error: Detected 20 oom-kill event(s) in StepId=824575.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
